decorator.rb

```diff
@@ -0,0 +1,25 @@
+require './nameable'
+require './person'
+
+class BaseDecorator < Nameable
+  def initialize(nameable)
+    super()
+    @nameable = nameable
+  end
+
+  def correct_name
+    @nameable.correct_name
+  end
+end
+
+class CapitalizeDecorator < BaseDecorator
+  def correct_name
+    @nameable.correct_name.capitalize
+  end
+end
+
+class TrimmerDecorator < BaseDecorator
+  def correct_name
+    @nameable.correct_name.size > 10 ? @nameable.correct_name[0..9] : @nameable.correct_name
+  end
```

Comment:
- [ ] Hi, Here is the outcome I am getting when I am checking your code.
![Capture](https://user-images.githubusercontent.com/63972670/223555738-660e0f79-d044-420e-9c48-7144c183c31b.JPG)
- [ ] I would suggest you create Capitalize file and Trimmer file,  then move the class you created to each particular file. I believe it will help to fix the issue.

Reply:
Hi @OLIPLICHE, thanks for the review.